.github/workflows/test.yml

```diff
@@ -12,6 +12,23 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    env:
+      SQLX_OFFLINE: true
+      DATABASE_URL: postgres://postgres:password@localhost:5432/test
```

Comment:
We shouldn't use `SQLX_OFFLINE` here if we also set up the database, right? Instead, we should add `cargo sqlx prepare --check` somewhere as a test step, to check that the `.sqlx` directory is up to date. And the DB will need to be migrated with `cargo sqlx migrate run` first.

Reply:
Are I see. I was under the impression that `.sqlx` is gonna be used as the source of trust. Will remove it

Reply:
The main point is to check on CI whether `.sqlx` matches the queries that we have in code, but for that we need a live DB.